src/main/java/com/google/devtools/build/lib/remote/blobstore/http/AbstractHttpHandler.java

```diff
@@ -29,11 +30,14 @@
 import java.nio.channels.ClosedChannelException;
 import java.util.List;
 import java.util.Map;
+import org.apache.commons.lang.StringUtils;
```

Comment:
We are trying to get rid of apache commons as a dependency in Bazel. It seems overkill to add a dependency to apache commons as a one liner.

Reply:
Alright, noted :+1: